core-processor/src/precharge.rs

```diff
@@ -233,10 +233,10 @@ impl ContextCharged<ForCodeMetadata> {
     ) -> PrechargeResult<ContextCharged<ForInstrumentedCode>> {
         self.charge_gas(
             PreChargeGasOperation::InstrumentedCode,
-            block_config
-                .costs
-                .read
-                .cost_for_with_bytes(block_config.costs.read_per_byte, code_len_bytes.into()),
+            block_config.costs.db_costs.read.cost_for_with_bytes(
```

Comment:
overall renaming looks nice this way 
```suggestion
            block_config.costs.db.read.with_bytes(
```